### PR TITLE
favicon icon appearance

### PR DESCRIPTION
**Pull request by vespyrs**
addressing this issue: https://gitlab.torproject.org/tpo/web/community/-/issues/135

### CODE REVIEW

File: templates/meta.html [hunk 1]
```diff
@@ -3,4 +3,8 @@
 <meta name="description" content="{{ _('Defend yourself against tracking and surveillance. Circumvent censorship.') }} | {{ this.title }}">
 <link rel="stylesheet" href="{{ '/static/css/bootstrap.css'|asseturl }}">
 <link rel="stylesheet" href="{{ '/static/fonts/fontawesome/css/all.min.css'|asseturl }}" rel="stylesheet">
+<!-- Standard favicon -->
+<link rel="icon" type="image/x-icon" href="{{ '/static/images/favicon/favicon.ico'|asseturl }}">
```
Comment (gusgustavo) [inline templates/meta.html:7]: The right path is: /static/images/favicon.ico

File: templates/meta.html [hunk 2]
```diff
@@ -3,4 +3,8 @@
...
+<!-- Recommended favicon format -->
+<link rel="icon" type="image/png" href="{{ '/static/images/favicon/favicon.png'|asseturl }}">
```
Comment (gusgustavo) [inline templates/meta.html:9]: The right path is: /static/images/favicon.png

**Review by gusgustavo — CHANGES_REQUESTED** [review]
Please fix favicons path.

**Comment by vespyrs** [comment]
Fixed the path for the favicons.

**Pull request merged**